config/passport.js

```diff
@@ -1,26 +1,22 @@
-const passport = require("passport");
-const LocalStrategy = require("passport-local").Strategy;
-const mongoose = require("mongoose");
-const User = mongoose.model("User");
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const { User } = require('./../models');
 
-passport.use(
-    new LocalStrategy(
-        {
-            usernameField: "user[email]",
-            passwordField: "user[password]"
-        },
-        function(email, password, done) {
-            User.findOne({ email: email })
-                .then(function(user) {
-                    if (!user || !user.validPassword(password)) {
-                        return done(null, false, {
-                            errors: { "email or password": "is invalid" }
-                        });
-                    }
-
-                    return done(null, user);
-                })
-                .catch(done);
+passport.use(new LocalStrategy(
+  {
+    usernameField: 'user[email]',
+    passwordField: 'user[password]'
+  },
+  function(email, password, done) {
+    User.findOne({ where: { email }})
+      .then(function(user) {
+        if (!user || !user.validPassword(password)){
+          return done(null, false, {
```

Comment:
No header returned here. Maybe that would be added when we get to authentication.

Reply:
Yes, I think so.